Bryce/update aws onboarding (#15002)
DataDog/documentation: content/en/integrations/guide/aws-manual-setup.md

{{< tabs >}}
{{% tab "Role delegation" %}}

To set up the AWS integration manually, create an IAM policy and IAM role in your AWS account, and configure the role with an AWS External ID generated in your Datadog account. This allows Datadog's AWS account to query AWS APIs on your behalf, and pull data into your Datadog account. The sections below detail the steps for creating each of these components, and then completing the setup in your Datadog account.

## Setup

### Generate an external ID

> 1. In the [AWS integration tile][6], click **Add AWS Account**, and then select **Manually**.

Reviewer: ```suggestion
1. In the [AWS Integration configuration page][6], click **Add AWS Account**, and then select **Manually**.
```
Should be new link: https://app.datadoghq.com/integrations/amazon-web-services

Author: Thanks, updated!

2. Select `Role Delegation` for the access type and copy the `AWS External ID`. For more information about the external ID, read the [IAM User Guide][1].

>   **Note: Do not close the integration tile or the Datadog site**, as this causes the external ID value to reset.

Reviewer 2: ```suggestion
   **Note: Do not close the integration tile or the Datadog site**, as this causes the external ID value to reset.
```

Author: Thanks for pointing this out -- just to be clear, is this meant to start the note on the next line? That was my initial intention, but I found that adding just this space didn't accomplish that. The updated branch should show the desired effect, so let me know if you think it looks alright!

### AWS IAM role for Datadog
Create an IAM role for Datadog to use the permissions defined in the IAM policy.

8. Create a new role in the AWS [IAM Console][3].
9. Select **AWS account** for the trusted entity type, and **Another AWS account**.
10. Enter `464622532012` as the `Account ID`. This is Datadog's account ID, and grants Datadog access to your AWS data.
11. Select **Require external ID** and enter the external ID copied in the [Generate an external ID](#generate-an-external-id) section. 
Ensure to leave `Require MFA` disabled. 
12. Click **Next**.
13. If you've already created the policy, search for it on this page and select it. Otherwise, click **Create Policy**, which opens in a new window, and follow the instructions from the previous section. 
14. Optionally, attach the <a href="https://console.aws.amazon.com/iam/home#policies/arn:aws:iam::aws:policy/SecurityAudit" target="_blank">AWS SecurityAudit Policy</a> to the role to use Datadog's [Cloud Security Posture Management product][4] (CSPM).
15. Click **Next**.
16. Give the role a name such as `DatadogIntegrationRole`, as well as an apt description. 
17. Click **Create Role**.

### Complete the setup in Datadog

> 18. Return to the AWS integration tile page in your Datadog account that you had open in another tab, and click the checkbox to confirm the Datadog IAM role was added to the AWS account.

Reviewer: ```suggestion
18. Return to the AWS Integration configuration page for manually adding an account in Datadog that you had open in another tab, and click the checkbox to confirm the Datadog IAM role was added to the AWS account.
```

19. Enter the account ID **without dashes**, for example: `123456789012`. Your Account ID can be found in the ARN of the role created for Datadog.
20. Enter the name of the role created in the previous section, and click **Save**.

> **Note:** The role name you enter in the integration tile is case sensitive and must exactly match the role name in AWS.

Reviewer 2: ```suggestion
    **Note:** The role name you enter in the integration tile is case sensitive and must exactly match the role name in AWS.
```

Author: Thanks for pointing this out as well, updated so that the note starts on a new line.

21. If there is a [Datadog is not authorized to perform sts:AssumeRole][5] error, follow the troubleshooting steps recommended in the UI, or read the [troubleshooting guide][5].

> 22. Wait up to 10 minutes for data to start being collected, and then view the out-of-the-box <a href="https://app.datadoghq.com/screen/integration/7/aws-overview" target="_blank">AWS Overview Dashboard</a> to see metrics sent by your AWS services and infrastructure.

Reviewer 2: Why not use a regular markdown link here? (and a few other places in this page)

Author: The HTML links on this page were a direct request from the PM, so that they would open new tabs by default. We had some internal discussion around this at the time, but couldn't come up with a way to have markdown links open in a new tab by default (without altering this behavior for the entire site). Let me know if you have any suggestions for that, otherwise I'll leave them as HTML for now.

> 

Reviewer: ^ This is for the line above. This shouldn't be there for access key govcloud/china setup since there is no external ID:

`For more details, see the [How to use an external ID when granting access to your AWS resources to a third party][1] AWS documentation.`

### Datadog

3. In the [AWS integration tile][2], click **Add AWS Account**, and then select **Manually**.
4. Select the **Access Keys (GovCloud or China Only)** tab.
5. Enter your `Account ID`, `AWS Access Key` and `AWS Secret Key`. Only access and secret keys for GovCloud and China are accepted.
6. Click **Save**.

> 7. Wait up to 10 minutes for data to start being collected, and then view the out-of-the-box <a href="https://app.datadoghq.com/screen/integration/7/aws-overview" target="_blank">AWS Overview Dashboard</a> to see metrics sent by your AWS services and infrastructure.

Reviewer 2: use regular markdown link?

Author: As above, this is a workaround to have the link open in a new tab.

### Datadog

3. In the [AWS integration tile][2], click **Add AWS Account**, and then select **Manually**.
4. Select the **Access Keys (GovCloud or China Only)** tab.
5. Enter your `Account ID`, `AWS Access Key` and `AWS Secret Key`. Only access and secret keys for GovCloud and China are accepted.
6. Click **Save**.
7. Wait up to 10 minutes for data to start being collected, and then view the out-of-the-box <a href="https://app.datadoghq.com/screen/integration/7/aws-overview" target="_blank">AWS Overview Dashboard</a> to see metrics sent by your AWS services and infrastructure.

[1]: https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html
[2]: https://app.datadoghq.com/integrations/amazon-web-services
{{% /tab %}}
{{< /tabs >}}

> ## Configuration tabs

Author: Got it, removed
